documentation/Installing-Clear-Containers-on-Ubuntu.md

```diff
@@ -43,7 +43,7 @@ cat << EOL | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
 [Service]
 ExecStart=
 ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-oci-runtime --default-runtime=cor
-EOL>>
+EOL
```

Comment:
Yep - I didn't write that bit, but I agree. In fact, all other files uses `EOF`, so I've changed to `EOF` for consistency.